project_management/workshop_advert.md

```diff
@@ -0,0 +1,56 @@
+## Boost your research reproducibility with Binder
+
+If you are looking for an easy way to share your code with others, without worrying about what software they have, then myBinder.org might be the solution for you. Binder allows you to share your software with others in the form of a single clickable link, making it easier for users and removing the need for you to worry about supporting lots of different platforms. 
```

Comment:
Suggested rewording: "without worrying about the computational environment they're running or installing a long list of requirements"